var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -35,7 +35,7 @@ class SuiteSparse(Package):
     version('4.5.3', '8ec57324585df3c6483ad7f556afccbd')
     version('4.5.1', 'f0ea9aad8d2d1ffec66a5b6bfeff5319')
 
-    variant('tbb', default=True, description='Build with Intel TBB')
```

Comment:
I do not know the details about TBB, but it seems it is hardware specific. (https://www.threadingbuildingblocks.org/) -- it may not work with IBM POWER cpus. 

If you think this should stay in default, I can use "~tbb" instead.  Please let me know.